src/clarity-angular/data/datagrid/render/column-resizer.ts

```diff
@@ -42,16 +42,18 @@ export class DatagridColumnResizer implements AfterViewInit, OnDestroy {
     }
 
     ngAfterViewInit() {
-        this.columnMinWidth = this.domAdapter.minWidth(this.columnEl);
         this.handleTrackerEl = this.dragDispatcher.handleTrackerRef.nativeElement;
-
         this.dragDispatcher.addDragListener();
         this.subscriptions.push(this.dragDispatcher.onDragStart.subscribe(() => this.dragStartHandler()));
         this.subscriptions.push(this.dragDispatcher.onDragMove.subscribe(($event) => this.dragMoveHandler($event)));
         this.subscriptions.push(this.dragDispatcher.onDragEnd.subscribe(() => this.dragEndHandler()));
     }
 
     dragStartHandler(): void {
+        if (!this.columnMinWidth) {
```

Comment:
Question for my understanding. How is this different than initializing the min-width on `ngAfterViewInit` since even that lifecycle method will be called only once, no? Is this change related to hideable columns?

Reply:
This `dragStartHandler()` is called only when a user drags the column. The problem was that `this.columnMinWidth` was getting `Nan` value because the datagrid was instantiating even when it's nested inside a hidden modal.